recipe/meta.yaml

```
@@ -24,6 +24,7 @@ source:
 build:
   number: {{ build_number }}
   skip: True  # [cuda_compiler_version == "11.2"]
+  skip: True  # [r_base != "4.3"]
```

Would it be possible to retry this?

```suggestion
```

---

Without this line, Windows will fail because it doesn't yet have R 4.3.

---

Ok so XGBoost needs at least R 4.3?

If so, could we rewrite this to just skip [R 4.1 used on Windows]( https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/a724f060a0629f6be7b16c06cdce06467788dee5/recipe/conda_build_config.yaml#L831-L833 )? That way it will work with R 4.4 when that migration comes around

```suggestion
  skip: True  # [r_base == "4.1"]
```

Also would suggest we move this `skip` to the `r-xgboost*` packages specifically. That way we don't skip Windows altogether and just skip R on Windows